### PR TITLE
Small code changes

### PR DESCRIPTION
**Pull request by cristaloleg**

### CODE REVIEW

File: conn.go [hunk 1]
```diff
@@ -838,10 +841,6 @@ func (c *Conn) writeClose(p []byte, cerr error) error {
 	return nil
 }
 
-func init() {
```
Comment (nhooyr) [inline conn.go]: I'd prefer this stays here as it's only used with Ping.

File: conn.go [hunk 2]
```diff
@@ -685,9 +689,8 @@ func (c *Conn) writeFrame(ctx context.Context, fin bool, opcode opcode, p []byte
 	case <-c.closed:
 		return n, c.closeErr
 	case c.setWriteTimeout <- context.Background():
+		return n, nil
```
Comment (nhooyr) [inline conn.go]: Should be reverted, was intentional to match the write timeout code above.

File: conn.go [hunk 3]
```diff
@@ -70,7 +74,7 @@ type Conn struct {
 	activeReader *messageReader
 	// readFrameLock is acquired to read from bw.
 	readFrameLock     chan struct{}
-	readClosed        int64
+	isReadClosed      int64
```
Comment (nhooyr) [inline conn.go]: This is 👍

File: conn.go (hunk 1 above)
Comment (cristaloleg) [reply conn.go]: ok

**Pull request closed without merging**